20250226-actors-pubsub.md

```diff
@@ -0,0 +1,158 @@
+# Actor PubSub Implementation Proposal
+
+- Author(s): @joshvanl
+
+## Overview
+
+This proposal introduces PubSub capabilities for Dapr Actors, enabling actors to subscribe to messages for their actor type.
+Additionally, actors of a that type have the ability to publish messages to that type, to be delivered to  particular actor IDs.
+A single designated PubSub component will be marked as the actor PubSub for that namespace, similar to how an actor state store is defined.
+There can be only one actor PubSub Component per namespace.
+Actors will be spawned, if they are not already, when they receive an actor PubSub message.
+
+## Background
+
+Today, Dapr provides PubSub functionality for applications but lacks native PubSub integration for actors.
+This limitation means that actors cannot communicate efficiently via publish-subscribe messaging patterns within their type.
+By enabling Actor PubSub, actors can publish messages to specific actor IDs, improving scalability and communication reliability.
```

Comment:
Circling back to my other comment, having a more dynamic subscription would enable an actor to far more easily subscribe to messages from other actor types altogether (huge for scenarios where an actor "manager" is responsible for cross-domain activities).

In light of line 24, I'll write a separate proposal.